diff --git a/node_modules/@medusajs/framework/dist/http/middlewares/ensure-publishable-api-key.js b/node_modules/@medusajs/framework/dist/http/middlewares/ensure-publishable-api-key.js
index 7b0b8c6..1234567 100644
--- a/node_modules/@medusajs/framework/dist/http/middlewares/ensure-publishable-api-key.js
+++ b/node_modules/@medusajs/framework/dist/http/middlewares/ensure-publishable-api-key.js
@@ -4,10 +4,10 @@
 const utils_1 = require("@medusajs/utils");
 async function ensurePublishableApiKeyMiddleware(req, _, next) {
     const publishableApiKey = req.get(utils_1.PUBLISHABLE_KEY_HEADER);
-    if (!(0, utils_1.isPresent)(publishableApiKey)) {
-        const error = new utils_1.MedusaError(utils_1.MedusaError.Types.NOT_ALLOWED, `Publishable API key required in the request header: ${utils_1.PUBLISHABLE_KEY_HEADER}. You can manage your keys in settings in the dashboard.`);
-        return next(error);
-    }
+    // if (!(0, utils_1.isPresent)(publishableApiKey)) {
+    //     const error = new utils_1.MedusaError(utils_1.MedusaError.Types.NOT_ALLOWED, `Publishable API key required in the request header: ${utils_1.PUBLISHABLE_KEY_HEADER}. You can manage your keys in settings in the dashboard.`);
+    //     return next(error);
+    // }
     let apiKey;
     const query = req.scope.resolve(utils_1.ContainerRegistrationKeys.QUERY);
     try {
@@ -21,7 +21,7 @@
                 "sales_channels_link.sales_channel_id",
             ],
             filters: {
-                token: publishableApiKey,
+                ...(publishableApiKey ? { token: publishableApiKey } : {}),
                 type: utils_1.ApiKeyType.PUBLISHABLE,
             },
         }, {
